accelerator/cli_info.h

```diff
@@ -89,6 +90,7 @@ static struct ta_cli_argument_s {
     {"mwm", optional_argument, NULL, MWM_CLI, "minimum weight magnitude"},
     {"seed", optional_argument, NULL, SEED_CLI, "IOTA seed"},
     {"cache", required_argument, NULL, CACHE, "Enable/Disable cache server. It defaults to off"},
+    {"socket", required_argument, NULL, SOCKET, "Set the socket name of initializing notification"},
```

Comment:
`socket` is confusing. It is dedicated to local socket communications.

Reply:
how about `notify_socket`?

Reply:
How about `ipc`?

Reply:
done